src/main/resources/jenkins/scm/api/Messages.properties

```diff
@@ -21,3 +21,5 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
 #
+
+SCMNavigator.Description=Default implementation
```

Comment:
:ant: meaningless to a user. Why not say something vague like

> A collection of SCM repositories.

that would stand some chance of being correct?